source/ceylon/time/Date.ceylon

```diff
@@ -32,6 +32,15 @@ shared interface Date
 
     "Returns the [[DateRange]] between this and given [[Date]]."
     shared formal DateRange rangeTo( Date other );
+    
+    "Two `Date`s are considered equals if they have the same 
```

Comment:
ahh, and theres also `hash` .. if Im going to provide default `equals` implementation then I need to provide hash too, right ?